packages/shared-auth/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@hebo/shared-auth",
```

Comment:
Rename package to @hebo/shared-api with an auth folder. There will be additional shared functionalities between API and Gateway.